presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -283,6 +284,11 @@ public SystemSessionProperties(
                         featuresConfig.isLegacyOrderBy(),
                         false),
                 booleanSessionProperty(
+                        SUPPORT_NULL_ON_LHS_OF_SEMI_JOIN,
```

Comment:
I don't think we need a feature toggle. If I recall correctly, the "optimization" was not only incorrect in the general case, but it was almost useless in practice.